versions/QmctngrQAt9fjpQUZr7Bx3BsXUcif52eZGTizWhvcShsjz/static/js/Detector.js

```diff
@@ -5,56 +5,50 @@
 
 Detector = {
 
-  canvas : !! window.CanvasRenderingContext2D,
```

Comment:
reverted changes in the versions folder to preserve the hashes. Thanks for catching this!